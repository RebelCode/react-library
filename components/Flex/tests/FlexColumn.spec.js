import React from "react";
import { render, cleanup } from "@testing-library/react";
import { FlexColumn } from "../index";

afterEach(cleanup);

describe("FlexColumn", () => {
  const Comp = (
    <FlexColumn align="flex-start">
      <a href="">Go to Posts</a>
      <a href="">Go to Posts</a>
    </FlexColumn>
  );

  it("should take a snapshot", () => {
    const { asFragment } = render(Comp)

    expect(asFragment(Comp)).toMatchSnapshot();
  });

  it("renders correctly", () => {
    const { queryByLabelText } = render(Comp);

    expect(queryByLabelText("FlexColumn")).toBeTruthy();
  });
});
