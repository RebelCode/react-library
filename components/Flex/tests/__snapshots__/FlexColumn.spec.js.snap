// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlexColumn should take a snapshot 1`] = `
<DocumentFragment>
  <div
    aria-label="FlexColumn"
    style="display: flex; flex-flow: column nowrap; justify-content: flex-start; align-items: flex-start;"
  >
    <a
      href=""
    >
      Go to Posts
    </a>
    <a
      href=""
    >
      Go to Posts
    </a>
  </div>
</DocumentFragment>
`;
